packages/frontend-2/components/server-management/Table.vue

```diff
@@ -0,0 +1,122 @@
+<template>
+  <div class="mt-8 text-foreground">
```

Comment:
Not a good practice to add outer margins to a component, cause you may want to use it in different kinds of layouts, some of which may not need that margin or may need a different kind of margin.

Better to add this margin to the template where you actually use this component (`<Table class="mt-8"/>`)